cmd/src/config_edit.go

```diff
@@ -34,6 +34,9 @@ Examples:
 
     	$ src config edit -subject=$(src orgs get -f '{{.ID}}' -name=abc-org) -overwrite -value '{"motd":["Hello!"]}'
 
+  Change global settings:
+		
+        $ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```

Comment:
The indentation on this was slightly off compared to the earlier examples:

```suggestion
      $ src config edit -subject=$(echo 'query { site { id } }' | src api | jq .data.site.id --raw-output)
```